db/data/20220803164723_fix_end_academic_cycle_id_for_awarded_withdrawn_trainees.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class FixEndAcademicCycleIdForAwardedWithdrawnTrainees < ActiveRecord::Migration[6.1]
+  def up
+    hesa_trainees = Trainee.imported_from_hesa.where("hesa_updated_at > ?", "01/06/2022".to_date)
```

Comment:
I've chosen to exclude HESA trainees that were updated from HESA since 01/06/2022 so we don't include any of the trainees we incorrectly awarded. I might not be using the best date though.

Reply:
I actually think maybe I should exclude all imported from HESA trainees - I've been trying to get around querying for a massive number of trainees